Framework/Muon/test/RemoveExpDecayTest.h

```diff
@@ -32,6 +33,10 @@ MatrixWorkspace_sptr createWorkspace(size_t nspec, size_t maxt) {
   MantidVec X;
   MantidVec Y;
   MantidVec E;
+  if (useBinEdges) {
+    X.emplace_back(0.0);
+    X.emplace_back(0.0);
```

Comment:
Thanks for making the changes. All looks good apart from one little thing related to these extra x values. 
I think this code is currently adding both of the extra x values on the front of the first histogram. Also it assumes nspec is always equal to 2 ie always two histograms. I think you should move a single X.emplace_back(0.0) statement down to in between lines 40 and 41 and then that will work better. You might have to adjust the test values lower down as well where you check the x values at index 10, 19 etc are what you expect.